README.md

```diff
@@ -48,62 +52,48 @@ This is what Timber's `.twig` files look like (from this [Hello World example](h
-
-```bash
-cd ~/wp-content/themes/my-theme
-composer require timber/timber
-```
-
-If your theme/plugin is not setup to pull in Composer’s autoload file, you will need to load the vendor dependencies.
-
-**functions.php**
-
-```php
-require_once __DIR__ . '/vendor/autoload.php';
-```
-
-After this line, initialize Timber with
-
-```php
-Timber\Timber::init();
-```
+Follow the [Installation Guide](https://timber.github.io/docs/v2/installation/) for how to install Timber using Composer.
 
 ### What Now?
 
-Setup the [Timber Starter Theme](https://timber.github.io/docs/getting-started/setup/#use-the-starter-theme). Once you have that installed in your WordPress setup, continue reading the [Getting Started guide to Themeing](https://timber.github.io/docs/getting-started/theming/).
+Setup the [Timber Starter Theme](https://timber.github.io/docs/getting-started/setup/#use-the-starter-theme). Once you have that installed in your WordPress setup, continue reading the [Getting Started Guide](https://timber.github.io/docs/v2/getting-started/introduction/).
 
-* * *
-
-### Mission Statement
+## Timber’s mission
 
 Timber is a tool for developers who want to translate their HTML into high-quality WordPress themes through an intuitive, consistent and fully-accessible interface.
 * **Intuitive**: The API is written to be user-centric around a programmer's expectations.
 * **Consistent**: WordPress objects can be accessed through common polymorphic properties like slug, ID and name.
 * **Accessible**: No black boxes. Every effort is made so the developer has access to 100% of their HTML.
 
-#### What does it look like?
+### What does it look like?
 Nothing. Timber is meant for you to build a theme on. Like [underscores](https://github.com/Automattic/_s) it comes style-free, because you're the style expert. Instead, Timber handles the logic you need to make a kick-ass looking site.
 
-#### Who is it good for?
+### Who is it good for?
 Timber is great for any WordPress developer who cares about writing good, maintainable code. It helps teams of designers and developers working together. At [Upstatement](http://upstatement.com) we made Timber because while our entire team needs to participate in building WordPress sites, not everyone knows the ins-and-outs of the_loop(),  codex and PHP (nor should they). With Timber your best WordPress engineer can focus on building the `.php` files with requests from WordPress and pass the data into `.twig` files. Once there, designers can easily mark-up data and build out a site's look-and-feel.
 
-#### Related & Official Projects
+### Should I use it?
+
+Timber is MIT-licensed, so please use in personal or commercial work. Just don't re-sell it. Timber is used on [tens of thousands of sites](https://www.upstatement.com/timber/#showcase) (and tons more we don't know about)
+
+## Links
```

Comment:
Links is still empty, what did you had in mind there?

Reply:
If you look at https://github.com/timber/timber/tree/2.x-prepare-release#links, you’ll see what I mean. I just put all the links for the related projects into a separate section with that "Link" heading.

Reply:
Ah gotcha!